src/legacy/core_plugins/kibana/public/dashboard/reducers/embeddables.test.ts

```diff
@@ -43,11 +43,11 @@ describe('embeddableIsInitializing', () => {
   test('clears the error', () => {
     store.dispatch(embeddableIsInitializing('foo1'));
     const initialized = getEmbeddableInitialized(store.getState(), 'foo1');
-    expect(initialized).toBe(false);
+    expect(initialized).toEqual(false);
```

Comment:
Is there a reason why this is changed from `toBe` to `toEqual`?

Reply:
Yep changed back to toBe in commits, having determined the change was unnecessary earlier

Reply:
Did these commits make it in?  Still see `toEqual` on the diff here.